<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditParamDialog</class>
 <widget class="QDialog" name="EditParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device Parameter Edit</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDeviceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>326</height>
    </rect>
   </property>
   <property name="title">
    <string>Device List</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>275</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation</string>
   </property>
   <widget class="QPushButton" name="pushButtonReadDeviceInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWrite">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemoveFormList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResetDevice">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetDevice</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Device Infamation</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Model:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDevcieModel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Model</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>411</width>
     <height>196</height>
    </rect>
   </property>
   <property name="title">
    <string>Device Parameter</string>
   </property>
   <widget class="QCheckBox" name="checkBoxDeviceName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DeviceName:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxGroup1Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GroupName1:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxGroup2Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GroupName2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDeviceName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGroupName1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGroupName2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxBroadcastPeriod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Broadcast period (S):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBroadcastTime">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLocalPort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Device Listion Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDeviceListionPort">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemoteIpAdddr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manger Host Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHostAddress">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemotePort">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UdpPort:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHostPort">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxChnagePassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>126</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>165</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>165</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPasswordAgain">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>165</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelStatusBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>status:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDeviceName</tabstop>
  <tabstop>lineEditGroupName1</tabstop>
  <tabstop>lineEditGroupName2</tabstop>
  <tabstop>lineEditBroadcastTime</tabstop>
  <tabstop>lineEditDeviceListionPort</tabstop>
  <tabstop>lineEditHostAddress</tabstop>
  <tabstop>lineEditHostPort</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditPasswordAgain</tabstop>
  <tabstop>pushButtonWrite</tabstop>
  <tabstop>pushButtonResetDevice</tabstop>
  <tabstop>pushButtonReadDeviceInfo</tabstop>
  <tabstop>pushButtonRemoveFormList</tabstop>
  <tabstop>checkBoxDeviceName</tabstop>
  <tabstop>checkBoxGroup1Name</tabstop>
  <tabstop>checkBoxGroup2Name</tabstop>
  <tabstop>checkBoxBroadcastPeriod</tabstop>
  <tabstop>checkBoxRemoteIpAdddr</tabstop>
  <tabstop>checkBoxChnagePassword</tabstop>
  <tabstop>checkBoxLocalPort</tabstop>
  <tabstop>checkBoxRemotePort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
